### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by dnsanche**
I was able to solve the problems. It is working now.

### CODE REVIEW

**Comment by rfeseha** [comment]
Hey Dani I have reviewed your pullrequest and I think it looks good. The question I had is In the DockerFile why is the entry point EventBriteAssignement instead of EventCatalogApi. The other one Is in the docker compose yml file why is the context EventbriteAssignment instead of EventCatalogApi.

File: EventBriteAssignment/Dockerfile [hunk 1]
```diff
@@ -4,16 +4,16 @@ EXPOSE 80
 
 FROM mcr.microsoft.com/dotnet/core/sdk:2.1-stretch AS build
 WORKDIR /src
-COPY ["EventBriteCatalog/EventBriteCatalog.csproj", "EventBriteCatalog/"]
-RUN dotnet restore "EventBriteCatalog/EventBriteCatalog.csproj"
+COPY ["EventBriteAssignment/EventBriteAssignment.csproj", "EventBriteAssignment/"]
```
Comment (vlau26) [inline EventBriteAssignment/Dockerfile]: We changed the name of the project to EventCatalog from 3A. The folder is still called EventBriteAssignment though
```suggestion
COPY ["EventBriteAssignment/EventCatalog.csproj", "EventBriteAssignment/"]
```

File: EventBriteAssignment/Dockerfile [hunk 2]
```diff
@@ -4,16 +4,16 @@ EXPOSE 80
...
+RUN dotnet restore "EventBriteAssignment/EventBriteAssignment.csproj"
```
Comment (vlau26) [inline EventBriteAssignment/Dockerfile]:
```suggestion
RUN dotnet restore "EventBriteAssignment/EventCatalog.csproj"
```

File: EventBriteAssignment/Dockerfile [hunk 3]
```diff
@@ -4,16 +4,16 @@ EXPOSE 80
...
 COPY . .
-WORKDIR "/src/EventBriteCatalog"
-RUN dotnet build "EventBriteCatalog.csproj" -c Release -o /app
+WORKDIR "/src/EventBriteAssignment"
+RUN dotnet build "EventBriteAssignment.csproj" -c Release -o /app
```
Comment (vlau26) [inline EventBriteAssignment/Dockerfile]:
```suggestion
RUN dotnet build "EventCatalog.csproj" -c Release -o /app
```

File: EventBriteAssignment/Dockerfile [hunk 4]
```diff
@@ -4,16 +4,16 @@ EXPOSE 80
...
 
 FROM build AS publish
-RUN dotnet publish "EventBriteCatalog.csproj" -c Release -o /app
+RUN dotnet publish "EventBriteAssignment.csproj" -c Release -o /app
```
Comment (vlau26) [inline EventBriteAssignment/Dockerfile]:
```suggestion
RUN dotnet publish "EventCatalog.csproj" -c Release -o /app
```

File: EventBriteAssignment/Dockerfile [hunk 5]
```diff
@@ -4,16 +4,16 @@ EXPOSE 80
...
 
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app .
-ENTRYPOINT ["dotnet", "EventBriteCatalog.dll"]
\ No newline at end of file
+ENTRYPOINT ["dotnet", "EventBriteAssignment.dll"]
```
Comment (vlau26) [inline EventBriteAssignment/Dockerfile]:
```suggestion
ENTRYPOINT ["dotnet", "EventCatalog.dll"]
```

**Pull request merged**